scientist-softserv/palni-palci
# add-back-ga

Ref ticket: https://github.com/scientist-softserv/palni-palci/issues/339
- Adding documentation to the README
- Uncommenting the values in the analytics.yaml file
- Update the web and worker script
- Update the Dockerfile to call the web script
- Adding the env variables to each deployment values file & .env

Testing Instructions:

- Navigate to a current tenant and see the ga areas of the app (Dashboard/Reports/etc) is working with no errors
- Create a new tenant and navigate to that tenant and see the ga areas of the app (Dashboard/Reports/etc) is working with no errors
